shell/browser/mac/electron_application_delegate.mm

```diff
@@ -69,9 +69,9 @@ - (void)applicationWillFinishLaunching:(NSNotification*)notify {
 
 - (void)applicationDidFinishLaunching:(NSNotification*)notify {
   NSUserNotification* user_notification =
-      [notify userInfo][(id) @"NSApplicationLaunchUserNotificationKey"];
+      [notify userInfo][NSApplicationLaunchUserNotificationKey];
 
-  if (user_notification.userInfo) {
+  if ([user_notification isKindOfClass:[NSUserNotification class]]) {
```

Comment:
The old code checked for `userInfo != null`, should we still be doing that?

Reply:
https://github.com/electron/electron/blob/822b04406873e60dcbccc29c5e8e980329d4b426/shell/browser/mac/dict_util.mm#L74

Reply:
Checking `userInfo` for `nil` is not necessary, as it's done by `NSDictionaryToDictionaryValue`.